app-backup/borgbackup/borgbackup-1.4.2-r1.ebuild

```diff
@@ -45,6 +45,11 @@ BDEPEND="
 EPYTEST_PLUGINS=()
 distutils_enable_tests pytest
 
+src_configure() {
+	# used by dev-python/pkgconfig
+	tc-export PKG_CONFIG
```

Comment:
In theory should call distutils-r1_src_configure but it also doesn't matter as it's purely for hooks defined in the ebuild.